src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/PSUserAgent.cs

```diff
@@ -122,15 +123,24 @@ internal static string Platform
         {
             get
             {
-                return ("Windows NT");
+                if(RuntimeInformation.IsOSPlatform(OSPlatform.Windows)){
```

Comment:
I suggest use `public static class Platform` from CorePsPlatform.cs and rename `Platform` in PSUserAgent.cs to PSUserAgentPlatform.

Also please use common formatting pattern:
```c#
if ( ... )
{
    ...
}
else if ( ... )
{
    ...
}
```